k8s/logdna-agent-st.yaml

```diff
@@ -123,6 +123,17 @@ spec:
               value: /supertenant/logs/ingest
             - name: RUST_LOG
               value: info
+            - name: LOGDNA_LOG_K8S_EVENTS
```

Comment:
Which yamls do I need to add this to?

Reply:
We might want to keep `logdna-agent.yaml` and `logdna-agent-st.yaml` unchanged, instead modifying `agent-resources.yaml` and `agent-resources-supertenant.yaml`

Reply:
I've added this (and the CAP_ stuff to all of the yamls as I don't think any of them would run without the CAP stuff.

Do you think I should remove the LOGDNA_LOG_K8S_EVENTS from logdna-agent.yaml and logdna-agent-st.yaml? I'm still not 100% clear on what each is used for

Reply:
I had to really think back to the purpose of this file. These files are for backwards compatibility for the YAML files we host on our asset server. Those files are generated from the github files here. Keeping the old files was in case we had lingering documentation which referenced those old links to `logdna-agent.yaml`.

That being the case it's probably best to keep this files "old".